PickMeUp/PickMeUp/PMEmbedTableViewController.m

```diff
@@ -13,20 +14,39 @@
 
 @interface PMEmbedTableViewController () <UITableViewDataSource, UITableViewDelegate>
 @property (weak, nonatomic) IBOutlet UITableView *tableView;
+@property (strong, nonatomic) CBStoreHouseRefreshControl *storeHouseRefreshControl;
 @end
 
 @implementation PMEmbedTableViewController
 
 static const NSString *const kGoToDetailsSegue = @"goToDetails";
 static const NSString *const kPostCellIdentifier = @"postCell";
+static const NSString *const kStoreHouseFile = @"storeHouse";
 
 - (void)viewDidLoad {
     [super viewDidLoad];
     self.tableView.dataSource = self;
     self.tableView.delegate = self;
     self.tableView.rowHeight = UITableViewAutomaticDimension;
+    self.storeHouseRefreshControl = [CBStoreHouseRefreshControl attachToScrollView:self.tableView target:self refreshAction:@selector(refreshTriggered:) plist:kStoreHouseFile color:[UIColor darkGrayColor] lineWidth:1.5 dropHeight:80 scale:1 horizontalRandomness:150 reverseLoadingAnimation:YES internalAnimationFactor:0.5];
```

Comment:
please set max line length guide to 120 in Xcode and make it should line end guides like I described in this [doc](https://docs.google.com/document/d/13fxjENsh87wuAR_aEwMdVKfGtIbeiaYCtEYgbxgvQHs/edit#heading=h.u7hp3ks8rwij)